modules/nodes-pods-vertical-autoscaler-uninstall.adoc

```diff
@@ -20,8 +20,13 @@ You can remove a specific VPA using the `oc delete vpa <vpa-name>` command. The
 
 . In the {product-title} web console, click *Operators* → *Installed Operators*.
 
-. Switch to the *openshift-vertical-pod-autoscaler* project. 
+. Switch to the *openshift-vertical-pod-autoscaler* project.
 
 . Find the *VerticalPodAutoscaler*  Operator and click the Options menu. Select *Uninstall Operator*.
 
 . In the dialog box, click *Uninstall*.
+
+[NOTE]
+====
+The Operator, any Operator deployments and pods are removed by this action. Any Operands, and resources managed by the Operator, including CRDs and CRs, are not removed. To remove these after uninstalling the Operator, you might need to manually delete them.
```

Comment:
Would this be better worked into the module description? Since it's a deletion action, does it makes sense to walk through what is deleted and what isn't prior to performing the procedure?

Reply:
> Not sure how the first sentence scans. Maybe "This action removes any Operator deployments and pods, as well as the Operator itself."
> 
> Do we typically capitalize "Operands?" Haven't seen that before.
> 
> "Might need?" When or when not?

This is an as-is sentence which was written originally in the other file. I can make this change as an active voice adheres to our guidelines and don't see an exception why this should be a passive one. 

"Operands" is a term similar to "Operator". Another reference in our document - https://docs.openshift.com/container-platform/4.8/operators/admin/olm-managing-custom-catalogs.html#olm-accessing-images-private-registries_olm-managing-custom-catalogs

Some users might be actively using CLI and would have written some robust cleanup logic, in which case, the manual deletion will not be needed. Up to user's discretion.  "might" is probably not conveying it well. I will replace it with "can" as it sounds more appropriate.

Reply:
> Would this be better worked into the module description? Since it's a deletion action, does it makes sense to walk through what is deleted and what isn't prior to performing the procedure?

Nice suggestion. Thank you.